extension/app/src/components/auth/useAuth.ts

```diff
@@ -63,11 +69,11 @@ export const useAuthHook = () => {
     const handleStorageChange = (changes: {
       [key: string]: chrome.storage.StorageChange;
     }) => {
-      if (changes.accessToken && !changes.accessToken.newValue) {
-        console.log("Access token removed from storage.");
-        setTokens(null);
-        setUser(null);
-      }
+      // if (changes.accessToken && !changes.accessToken.newValue) {
+      //   console.log("Access token removed from storage.");
+      //   setTokens(null);
+      //   setUser(null);
+      // }
```

Comment:
I had to comment this - otherwise we're unlogged when the token change (so on rfresh)

Reply:
Very weird because this checks that we do not receive a new value of the token (no `newValue`)